test/features/support/GithubAPI/Issues/IssuesTestSteps.js

```diff
@@ -0,0 +1,39 @@
+const defaults = require('superagent-defaults');
+
+const agent = defaults();
+const { Given, When, Then } = require('cucumber');
+const chai = require('chai');
+const { setAuthorizationHeaders } = require('../../helpers/helpers');
+
+const { expect } = chai;
+
+const urlBase = 'https://api.github.com';
+
+let githubRepositoryName;
+
+const query = { title: 'API Created Issue' };
+const updateQuery = { body: 'API Created Issue new body' };
+
+Given('a public repository of the account {string}', (repository) => {
+  setAuthorizationHeaders(agent);
+  githubRepositoryName = repository;
+});
+
+When('a request is used to create an issue in the accounts repository', async function () {
+  this.response = await agent.post(`${urlBase}/repos/${this.username}/${githubRepositoryName}/issues`).send(query);
+});
+
+Then('the issue should have been created', function () {
+  expect(this.response.body.id).to.not.equal('undefined');
```

Comment:
That assertion is not recommended. (https://www.chaijs.com/api/bdd/#method_undefined)
May you prefer to validate that that property is a string or follows a specific structure